code/game/objects/items/plushes.dm

```diff
@@ -33,7 +32,8 @@
 
 /obj/item/toy/plush/Initialize()
 	. = ..()
-	AddComponent(/datum/component/squeak, squeak_override)
+	for(var/A in datum_outputs)
+		AddComponent(/datum/component/squeak, A)
```

Comment:
what the fuck

Reply:
so that if plushes have multiple datum outputs all of them play, it's a list after all